Wikipedia/Images/WMFImageController.swift

```diff
@@ -124,14 +123,54 @@ public class WMFImageController : NSObject {
         .then(mainImageBlock)
     }
 
-    /// MARK: - Simple Fetching
+    /**
+    Use the shared session to download an image directly to the receiver's disk cache.
+
+    - parameter url: The URL to download the image from.
+
+    - returns: A promise which is resolved after the data has been successfully written to disk.
+    */
+    public func downloadImageDataInBackground(url: NSURL) -> Promise<Void> {
```

Comment:
Some thoughts on the name here:
- It's not image data, its the image right?
- By background you mean low priority? Maybe we should say low priority - background just sounds like a background thread.
- The fact that this bypasses the memory cache seems really important.
- Other methods say `fetch`, not `download`

With this in mind something like `fetchImageWithLowPriorityAndCacheToDisk` or `fetchImageWithLowPriorityBypassingDiskCache` or `fetchImageWithLowPriority(url: NSURL cacheToDisk:BOOL cacheInMemory:BOOL)` seems more appropriate.